### PR TITLE
Update logic for pulling down roadmap file

### PR DESCRIPTION
**Pull request by joebochill**
For PXBLUE-1905

#### Changes proposed in this Pull Request:
- Pull a new roadmap file each time user changes the dropdown
- Remove the 'All' option
- Add options for R21

### CODE REVIEW

File: src/app/pages/Roadmap.tsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ export const Roadmap: React.FC = (): JSX.Element => {
     const [typeFilter, setTypeFilter] = useState<ItemTypeFilter>('all');
     const [statusFilter, setStatusFilter] = useState<Status | 'all'>('all');
     const [frameworkFilter, setFrameworkFilter] = useState<FrameworkFilter>('all');
-    const [releaseFilter, setReleaseFilter] = useState<Release | 'all'>('R19');
+    const [releaseFilter, setReleaseFilter] = useState<Release>('R20');
```
Comment (huayunh) [inline src/app/pages/Roadmap.tsx:113]: I wish this string is exported from the configuration folder...

File: src/app/pages/Roadmap.tsx [hunk 2]
```diff
@@ -152,7 +152,7 @@ export const Roadmap: React.FC = (): JSX.Element => {
     const clearFilters = useCallback(() => {
         setTypeFilter('all');
         setFrameworkFilter('all');
-        setReleaseFilter('all');
+        setReleaseFilter('R20');
```
Comment (huayunh) [inline src/app/pages/Roadmap.tsx:155]: So that this can get automatically set too

**Review by huayunh — APPROVED** [review]

**Comment by joebochill** [comment]
@huayunh like this?

**Review by jeffvg — APPROVED** [review]
looks good

**Comment by huayunh** [comment]
Looks good

**Pull request merged**